File-Formats/PDF/Convert-HTML-To-PDF/linux.md

```diff
@@ -7,104 +7,96 @@ documentation: UG
 keywords: Assemblies
 ---
 
-## Ubuntu
+# Ubuntu
 
 The Syncfusion HTML to PDF converter is a .NET library that converts HTML or web pages to PDF.  Using this library, convert an HTML to PDF in Linux.
```

Comment:
The Syncfusion HTML to PDF converter is a .NET library that converts HTML or web pages to PDF document in Linux. - Modify like this

Reply:
modified